css/index.css

```diff
@@ -177,37 +192,38 @@ header nav a:hover {
 }
 
 /* Desktop Header */
-@media (min-width: 769px) {
-  .container {
-    width: 880px;
-  }
 
+@media (min-width: 769px) {
```

Comment:
Love seeing the implementation for min-width for this project. They definitely come in handy and stops the cascading from spilling over to a section you would not want.